### ISSUE TITLE
 Test wmts:Server.KVP.GET.GetCapabilities.Response.SchemaReferenced does not detect valid schema reference

### ISSUE THREAD

**Issue by robinhoutmeyers** [opengeospatial/ets-wmts10#42]
The test verifies whether the GetCapabilities response entity references a normative(wmts.xsd or wmtsGetCapabilities_response.xsd) XML schema.

This is the case with the tested service http://demo.luciad.com:8080/LuciadFusionPlatform/ogc/wmts/bluemarbleworldimage?service=WMTS&request=GetCapabilities (it refers to wmts.xsd on schemas.opengis.net) but the test does not seem to detect this.

We get a test failure:

Message d22129e5038_1:
   Capabilities document references http://schemas.opengis.net/wmts/1.0/wmts.xsd http://www.opengis.net/ows/1.1 http://schemas.opengis.net/ows/1.1.0/owsAll.xsd http://www.w3.org/2001/XMLSchema http://www.w3.org/2001/XMLSchema.xsd http://www.w3.org/1999/xlink http://www.w3.org/1999/xlink.xsd  which is not a normative schema: wmts.xsd,wmtsGetCapabilities_response.xsd

Result: Failed

**Comment by dstenger** [comment]
Thank you for reporting. We will do further investigation.

**Comment by keshav-nangare** [comment]
Fixed with this commit: ce2df81d8250e913789b3e6e6393dbcc09469b6c. [PR #43]

**Comment by dstenger** [comment]
@robinhoutmeyers The provided test service is not available any more. Can you please reactivate it to enable testing of this fix?

**Comment by robinhoutmeyers** [comment]
@dstenger yes, I'll enable it Monday morning (currently out of office without access to the server).

**Comment by robinhoutmeyers** [comment]
@dstenger You can access the capabilities again at http://demo.luciad.com:8080/LuciadFusionPlatformT/ogc/wmts/bluemarble?service=WMTS&request=GetCapabilities

**Comment by dstenger** [comment]
@robinhoutmeyers 

Thank you for providing the service.
I tested the fix successfully using your service and merged the pull request.

We will create a new release and install it on beta web site soon.
